common/qt_probing.py

```diff
@@ -101,7 +101,6 @@
     #     os.seteuid(1000)
     #     logger.debug(f"New euid: {os.geteuid()}")
 
-    from PyQt6 import QtCore
```

Comment:
Don't delete this line. It is inside a try-block. The purpose here is to provoke an ImportError. That probing script is a workaround from "aryoda".

Add this comment line before that import to disable pylint for that line:

    # pylint: disable-next=unused-import

Reply:
Done